debian/kolibri.scripts-common

```diff
@@ -264,6 +264,14 @@ kolibri_configure()
         fi
         addgroup --system "$new_group_name"
         usermod -g "$KOLIBRI_USER" "$new_group_name"
+
+        # If a ~/.kolibri already existed, then set group ownership
+        DEFAULT_KOLIBRI_HOME="$(getent passwd $KOLIBRI_USER | awk -F ':' '{print $6}')/.kolibri"
+        if [ -e "$DEFAULT_KOLIBRI_HOME" ]
+        then
+            echo "Changing group ownership on $DEFAULT_KOLIBRI_HOME"
+            chgrp --recursive "$new_group_name" "$DEFAULT_KOLIBRI_HOME"
+        fi
```

Comment:
this change wasn't mentioned in the PR description or changelog. was it intentionally included?

Reply:
I added it because it was already in the current package but not in our git code

Reply:
The change is in this PR: https://github.com/learningequality/kolibri-installer-debian/pull/65/files, but it's not merged yet.

Reply:
it's not merged but it's in the package that's been released.

Reply:
I have removed that code, but I do think PR 65 should be merged if it has been already included in the latest releases.

Reply:
great thank you

let's try to only release packages from master moving forward